### ISSUE TITLE
Accessibility - About this App Screen - TalkBack Does Not Read Links as Links

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor-Android#26]
TalkBack does not read the ‘API’ or ‘www.dol.gov.ilab’ links as a link. 

Also TalkBack does not read ‘GlobalKids@dol.gov as a link so a person can send an e-mail to them. 

Not sure if the little bell sounds is what is supposed to tell a person that certain text is a link.

**Comment by aquilent-dgemoets** [comment]
This is currently how TalkBack reads HTML links and email addresses in standard Android WebViews.

**Comment by jfc3-dol** [comment]
Okay, this could be confusing for those using iOS at work and then Android at home or vice versa.
